dvc/command/experiments.py

```diff
@@ -304,7 +309,7 @@ def experiments_table(
 
     from dvc.compare import TabularData
 
-    headers = ["Experiment", "rev", "queued", "typ", "Created", "parent"]
+    headers = ["Experiment", "rev", "typ", "Created", "parent", "State"]
```

Comment:
I am a bit confused about what it should be: `State` or a `Status`. Both seem to make sense to me. Is `State` most appropriate here?

Reply:
I think either term works here. I just used `state` because `status` has some other connotations in DVC